src/views/Region.vue

```diff
@@ -256,10 +260,16 @@ export default {
     </div>
   </div>
   <div v-else class="container-fluid">
-    <search-box></search-box>
-
+    <b-navbar class="py-0" type="light" variant="light" fixed="top">
+      <b-button class="btn btn-sm btn-secondary mr-3" style="pointer-events: none;">Go to:</b-button>
+      <b-button @click="goto('region-plot')" class="mr-2" size="sm">Plot <span class="fas fa-level-down-alt"></span></b-button>
+      <b-button @click="goto('external-links')" class="mr-2" size="sm">External links <span class="fas fa-level-down-alt"></span></b-button>
+      <b-navbar-nav class="ml-auto">
```

Comment:
In practice, I suspect that we won't get many mobile users, so, not urgent.

But useful to know: On a small screen (like mobile device) the search bar disappears off the edge of the screen. Utilities like `b-collapse` can help this by creating defined regions that will appear in dropdown menus.
https://bootstrap-vue.js.org/docs/components/navbar

Similarly, layouts can specify "fraction of allowed width" (like col-sm-4) instead of pixel widths, which allow the search box to take up an appropriate amount of space on mobile without scrolling.

Reply:
I used pixel widths to ensure the placeholder text doesn't get cut off too much, so it's mostly a conscious choice even if it's sub-optimal for small screens.

I updated the placeholder text and simplified the home screen.